samza-rest/src/test/java/org/apache/samza/monitor/TestLocalStoreMonitor.java

```diff
@@ -65,7 +65,7 @@ public void setUp() throws Exception {
 
     // Initialize the local store monitor with mock and config
     localStoreMonitor = new LocalStoreMonitor(new LocalStoreMonitorConfig(new MapConfig(config)),
-                                              new NoOpMetricsRegistry(),
+                                              new LocalStoreMonitorMetrics("TestMonitorName", new NoOpMetricsRegistry()),
```

Comment:
Maybe NoOpMetricsRegistry is no longer appropriate and the new metrics should be tested.